### PR TITLE
web/storagenode: fix bandwidth graph

### LINKED ISSUE #4951: Storagenode dashboard: bandwidth graph disappears on click egress or ingress
storj: v1.57.2 (and v1.56.4 same)
bandwidth graph disappears on click egress or ingress

image

click egress:

image

Sometimes the graph looks strange

image

### PR DESCRIPTION
**Pull request by CatInAHatIsBack**
What: 

Fixes the bandwidth graph on the storagenode dashboard

Made earlier pr but messed something up with merge so made new pr

- https://github.com/storj/storj/pull/4990/commits/42fc2c8af306b102062236e5c27fc9b59e528066

Why:

Resolves #4951

Please describe the tests:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

**Comment by CatInAHatIsBack** [comment]
signed CLA

**Comment by CatInAHatIsBack** [comment]
does not seem like it was just me. "cockroach integration failed"

**Comment by Erikvv** [comment]
Yeah it's not related to your change. We'll take a look.

**Comment by Erikvv** [comment]
@stefanbenten can you check if CLA is OK now?

**Comment by CatInAHatIsBack** [comment]
Signed multiple cla's.

**Comment by profclems** [comment]
> Signed multiple cla's.

@CatInAHatIsBack, for legal reasons, it is required that you provide legitimate details when signing the CLA. We manually review the CLA before adding your username on file to be recognised by the @cla-bot. 
Your CLA submissions are quite suspicious and likely fake. This is a great PR that fixes an important issue on the storagenode dashboard and I'd love to merge this but I can't merge it if you don't sign the CLA with your correct details

**Comment by CatInAHatIsBack** [comment]
@profclems, the first cla's i signed did not contain real address or name. i have signed multiple cla's at later dates with real name and family members address. I could sign another, but i have already signed multiple with legitimate details

**Comment by profclems** [comment]
@CatInAHatIsBack the name used to sign the CLA is the name of an old historical journalist and the address is flagged in various places online.

**Comment by CatInAHatIsBack** [comment]
> Thank you for your pull request and welcome to our community. We require contributors to sign our [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLSdVzD5W8rx-J_jLaPuG31nbOzS8yhNIIu4yHvzonji6NeZ4ig/viewform), and we don't seem to have the users @CatInAHatIsBack on file. Once you have signed the CLA, please let us know, so we can manually review and add you to the approved contributors list.

Already signed the CLA

File: web/storagenode/src/app/components/BaseChart.vue [hunk 1]
```diff
@@ -31,6 +31,13 @@ export default class BaseChart extends Vue {
         chartContainer: HTMLElement;
     };
 
+    /**
+     * Rebuilds chat after type switch.
```
Comment (azorpax) [inline web/storagenode/src/app/components/BaseChart.vue:35]: misspelled chart

**Review by profclems — DISMISSED** [review]
LGTM! Thanks a lot for this PR, @CatInAHatIsBack

**Comment by profclems** [comment]
@cla-bot check

File: web/storagenode/src/app/components/BaseChart.vue (hunk 1 above)
Comment (amwolff) [inline web/storagenode/src/app/components/BaseChart.vue:35]:
```suggestion
     * Rebuilds chart after type switch.
```

**Review by amwolff — APPROVED** [review]

**Review by amwolff — APPROVED** [review]

**Review by profclems — APPROVED** [review]

**Pull request merged**